pkg/koordlet/metriccache/metric_types.go

```diff
@@ -36,11 +36,12 @@ const (
 type MetricKind string
 
 const (
-	NodeMetricCPUUsage     MetricKind = "node_cpu_usage"
-	NodeMetricMemoryUsage  MetricKind = "node_memory_usage"
-	NodeMetricGPUCoreUsage MetricKind = "node_gpu_core_usage"
-	NodeMetricGPUMemUsage  MetricKind = "node_gpu_memory_usage"
-	NodeMetricGPUMemTotal  MetricKind = "node_gpu_memory_total"
+	NodeMetricCPUUsage           MetricKind = "node_cpu_usage"
+	NodeMetricMemoryUsage        MetricKind = "node_memory_usage"
+	NodeMemoryWithPageCacheUsage MetricKind = "node_memory_with_page_cache_usage"
```

Comment:
recommend `NodeMemoryUsageWithPageCache` and `node_memory_usage_with_page_cache`